Terminal.Gui/Views/RuneCellEventArgs.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+
+namespace Terminal.Gui {
+	/// <summary>
+	/// Args for events that relate to a specific <see cref="RuneCell"/>.
+	/// </summary>
+	public class RuneCellEventArgs {
+		/// <summary>
+		/// The list of runes the RuneCell is part of.
+		/// </summary>
+		public List<RuneCell> Line { get; }
+
+		/// <summary>
+		/// The index of the RuneCell in the line.
+		/// </summary>
+		public int Col { get; }
+
+		/// <summary>
+		/// The unwrapped row and col index position in the text.
```

Comment:
Can we be more clear on what "unwrapped" means? 

Would this be accurate?

"The unwrapped row and column index into the text containing the RuneCell. Unwrapped means the text without word wrapping or other visual formatting having been applied."

Reply:
Thanks for the help. Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/5b08f1f0ef85060762b9f85169c93fb7543cbec2.